src/modules/previewpane/powerpreview/dllmain.cpp

```diff
@@ -0,0 +1,233 @@
+#include "pch.h"
+#include <interface/powertoy_module_interface.h>
+#include <interface/lowlevel_keyboard_event_data.h>
+#include <interface/win_hook_event_data.h>
+#include "trace.h"
+#include <../common/settings_objects.h>
+
+extern "C" IMAGE_DOS_HEADER __ImageBase;
+
+BOOL APIENTRY DllMain(HMODULE hModule, DWORD ul_reason_for_call, LPVOID lpReserved)
+{
+    switch (ul_reason_for_call)
+    {
+    case DLL_PROCESS_ATTACH:
+        Trace::RegisterProvider();
+        break;
+    case DLL_THREAD_ATTACH:
+    case DLL_THREAD_DETACH:
+        break;
+    case DLL_PROCESS_DETACH:
+        Trace::UnregisterProvider();
+        break;
+    }
+    return TRUE;
+}
+
+// PowerToy Winodws Explore File Preview Settings.
+struct PowerPreviewSettings
+{
+    bool svgPreview_in_explr_IsEnabled = true;
```

Comment:
nit: Since we are currently working on svg and md rendering in preview pane. I think we can later on add the explorer svg icon preview,